### ISSUE TITLE
Must be executed under progress indicator: com.intellij.openapi.progress.EmptyProgressIndicator@7309783b

### ISSUE THREAD

**Issue by sibelius** [jimkyndemeyer/js-graphql-intellij-plugin#109]
```java
Must be executed under progress indicator: com.intellij.openapi.progress.EmptyProgressIndicator@7309783b. Please see e.g. ProgressManager.runProcess()
java.lang.Throwable: Must be executed under progress indicator: com.intellij.openapi.progress.EmptyProgressIndicator@7309783b. Please see e.g. ProgressManager.runProcess()
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:136)
	at com.intellij.openapi.progress.impl.CoreProgressManager.assertUnderProgress(CoreProgressManager.java:734)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.processFilesConcurrentlyDespiteWriteActions(PsiSearchHelperImpl.java:343)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.processPsiFileRoots(PsiSearchHelperImpl.java:318)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.processElementsWithTextInGlobalScope(PsiSearchHelperImpl.java:293)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.bulkProcessElementsWithWord(PsiSearchHelperImpl.java:185)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.processElementsWithWord(PsiSearchHelperImpl.java:166)
	at com.intellij.psi.impl.search.PsiSearchHelperImpl.processElementsWithWord(PsiSearchHelperImpl.java:145)
	at com.intellij.lang.jsgraphql.ide.project.JSGraphQLPsiSearchHelper.resolveFragmentReference(JSGraphQLPsiSearchHelper.java:76)
	at com.intellij.lang.jsgraphql.psi.JSGraphQLNamedTypePsiElement.getReference(JSGraphQLNamedTypePsiElement.java:73)
	at com.intellij.lang.jsgraphql.psi.JSGraphQLNamedTypePsiElement.isErrorInContext(JSGraphQLNamedTypePsiElement.java:146)
	at com.intellij.lang.jsgraphql.ide.annotator.JSGraphQLAnnotator.apply(JSGraphQLAnnotator.java:120)
	at com.intellij.lang.jsgraphql.ide.annotator.JSGraphQLAnnotator.apply(JSGraphQLAnnotator.java:44)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.b(ExternalToolPass.java:191)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.access$700(ExternalToolPass.java:47)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.a(ExternalToolPass.java:165)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1132)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.run(ExternalToolPass.java:161)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:337)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:327)
	at com.intellij.util.ui.update.MergingUpdateQueue.lambda$flush$1(MergingUpdateQueue.java:277)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:291)
	at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:246)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:246)
	at com.intellij.util.Alarm$Request.runSafely(Alarm.java:417)
	at com.intellij.util.Alarm$Request.access$700(Alarm.java:344)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:384)
	at com.intellij.util.Alarm$Request.run(Alarm.java:395)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:242)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:212)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

WebStorm 2017.3 EAP
Build #WS-173.3302.13, built on October 18, 2017
WebStorm EAP User
Expiration date: November 17, 2017
JRE: 1.8.0_152-release-1024-b5 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13.1

**Comment by jimkyndemeyer** [comment]
Hi Sibelius.

I'm unable to reproduce this with the latest Webstorm EAP.

Can you verify that this is still an issue, and if so the specific steps to reproduce.

Thanks,
Jim.

**Comment by sibelius** [comment]
not anymore, it is fine

tks

**Comment by jimkyndemeyer** [comment]
Fixed in v1.62 for #115
